src/index.js

```diff
@@ -548,6 +546,21 @@ const indy = {
     return [revocRegId, JSON.parse(revocRegDelta), timestamp]
   },
 
+  async buildGetRevocRegRequest(
+    submitterDid: Did | null,
+    revocRegDefId: string,
+    timestamp: number
+  ): Promise<LedgerRequest> {
+    return JSON.parse(await IndySdk.buildGetRevocRegRequest(submitterDid, revocRegDefId, timestamp))
+  },
+
+  async parseGetRevocRegResponse(getRevocRegResponse: string): Promise<[RevRegId, RevocReg, number]> {
+    const [revocRegId, revocReg, ledgerTimestamp] = await IndySdk.parseGetRevocRegResponse(
+      JSON.stringify(getRevocRegResponse)
```

Comment:
`getRevocRegResponse` must be of type string, but is also stringified?

Reply:
Ahh whooops--good catch. Pushed up & fixed.